### PR TITLE
doc: document opts.skipCache for client.get

### PR DESCRIPTION
**Pull request by Nuhvi**

### CODE REVIEW

File: types/lib/client/index.d.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ declare class Client {
     _supscriptions: Map<string, () => void>;
     _sentPending: Promise<void>;
     get key(): any;
-    get id(): string;
+    get id(): any;
```
Comment (MiguelMedeiros) [inline types/lib/client/index.d.ts:40]: Saw the `any` type here. Any reason for that? Might be better with a specific type if possible. Thoughts?

File: types/lib/client/index.d.ts [hunk 2]
```diff
@@ -107,8 +111,8 @@ declare class Client {
      */
     _parseURL(url: string): Partial<{
         protocol: string;
-        key: Uint8Array;
-        id: string;
+        key: any;
+        id: any;
```
Comment (MiguelMedeiros) [inline types/lib/client/index.d.ts:115]: Same thing here with type: `any`

File: types/lib/client/index.d.ts (hunk 1 above)
Comment (MiguelMedeiros) [reply types/lib/client/index.d.ts:40]: It also seams that this is not related to the scope of this PR. 
Maybe could be a good idea to move this code to another PR.

**Review by MiguelMedeiros — CHANGES_REQUESTED** [review]

**Review by MiguelMedeiros — APPROVED** [review]
ACK

**Comment by rbndg** [comment]
@Nuhvi can you pls resolve conflicts

**Pull request merged**